src/website/app/pages/Styling/styling.md

```diff
@@ -0,0 +1,100 @@
+# Styling
+
+Mineral UI is built on a design system with styles ready to go out of the box.  We realize however that there will be cases when you need to customize styles for you own unique needs.  Below are some different techniques for customizing Mineral UI styles across all levels of your application.
```

Comment:
“customize styles for _your_ own unique”. I also think“own” can be omitted.

Reply:
Will update.